kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/CustomerRepository.kt

```diff
@@ -20,12 +20,17 @@ interface CustomerRepository {
 
 class JdbcCustomerRepository(private val db: Database) : CustomerRepository {
   init {
-    transaction(db) { SchemaUtils.create(Customers) }
+    transaction(db) {
+      SchemaUtils.create(Customers)
+      val missingColumnsStatements = SchemaUtils.addMissingColumnsStatements(Customers)
+      missingColumnsStatements.forEach { exec(it) }
+    }
   }
 
   override fun get(id: String): Customer? =
     transaction(db) {
-        Customers.select { Customers.id.eq(id) }
+        Customers.selectAll()
```

Comment:
Why not `SELECT * WHERE id=?`?

This seems inefficient.

Reply:
The old query builder is deprecated. This is the new way of writing the same query.

Reply:
The new design of the `exposed` looks weird to me.